cmake/blob_path.cmake

```diff
@@ -16,7 +16,6 @@ if(TARGET_OS MATCHES linux)
     -DPLATFORM_IV=\"${BLOB_PATH}/data/platform_iv.bin\"
     -DPLATFORM_HMAC_KEY=\"${BLOB_PATH}/data/platform_hmac_key.bin\"
     -DPLATFORM_AES_KEY=\"${BLOB_PATH}/data/platform_aes_key.bin\"
-    -DEPID_PRIVKEY=\"${BLOB_PATH}/data/epidprivkey.dat\"
     -DFDO_CRED=\"${BLOB_PATH}/data/PMDeviceCredentials.bin\"
```

Comment:
Seems like FDO_CRED isn't used anywhere either. Please remove this as well, here and Line 105.

Reply:
Updated in 4d168a5817db6fc30224708a431dee89d01fe254